Demo/Common/Networking/SessionRequest.swift

```diff
@@ -45,6 +45,7 @@ internal struct SessionRequest: APIRequest {
         //                                  "visa": InstallmentOptions(monthValues: [3, 6, 9], includesRevolving: true)]
         //
         //        try container.encode(installmentOptions, forKey: .installmentOptions)
+        //        try container.encode(true, forKey: .showInstallmentAmount)
```

Comment:
Maybe instead of commenting code, we can add "instalments feature" to demo app configuration ?

Reply:
I mean that's what the commented code says above right? :D
also it may not be worth it to add to settings anyway as the object is quite detailed for a settings area.

Reply:
ced1a5759c9d4979d02c50915d740df5d64d714e